core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -672,10 +651,6 @@ interface ImportDestinationArguments {
    * Bulk import the files in a directory into a table. Files can be created using
    * {@link RFile#newWriter()}.
    * <p>
-   * This new method of bulk import examines files in the current process outside of holding a table
```

Comment:
We can still document that files are examined client side w/o mentioning old bulk import.  Maybe something like.

```
The files are examined on the client side to determine destination tablets.  This examination will use memory and cpu within the process calling this API.
```

Reply:
Added text to javadoc in 6cd0b0e